packages/cfd/src/Containers/investor-password-manager.tsx

```diff
@@ -28,12 +29,14 @@ const InvestorPasswordManager = ({
     setPasswordType,
     toggleModal,
     validatePassword,
-}) => {
+}: TInvestorPasswordManager) => {
     if (is_submit_success_investor) {
         return <CFDPasswordSuccessMessage toggleModal={toggleModal} is_investor />;
     }
 
-    const initial_values = { old_password: '', new_password: '', password_type: 'investor' };
+    const type_investor = 'investor';
+
+    const initial_values = { old_password: '', new_password: '', password_type: `${type_investor}` };
```

Comment:
I had to initialize the value of password_type by a variable to avoid SonarCloud security Hotspot.